--- Amaya/amaya/Makefile.in.orig	Thu Apr 24 13:27:32 2003
+++ Amaya/amaya/Makefile.in	Thu Apr 24 13:27:55 2003
@@ -505,8 +505,6 @@
 	@(if test ! -d $(bindir) ; then $(MKDIR) $(bindir) ; fi)
 	$(INSTALL_BIN) ../bin/amaya $(prefix)/Amaya/applis/bin
 @NOPRINT@	$(INSTALL_BIN) ../bin/print $(prefix)/Amaya/applis/bin
-	$(RM) $(bindir)/amaya
-	$(LN_S) $(prefix)/Amaya/applis/bin/amaya $(bindir)/amaya
 	(for i in $(AMAYAFILES) ; do \
 	  for file in `$(ECHO) $(THOTDIR)/amaya/$$i` ; do \
 	    $(INSTALL_DATA) $$file $(prefix)/Amaya/amaya ; \
